src/style.css

```diff
@@ -0,0 +1,124 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  padding: 32px 16px 15px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  display: flex;
+  margin-bottom: 24px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.star {
+  display: flex;
+  text-align: center;
+}
+
+.stars {
+
+  background: url(\images\star.svg) no-repeat center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  border-radius: 0.5px;
+}
+
+.stars--4 .stars:nth-child(-n+4) {
+  background: url(images\star-active.svg) no-repeat center;
+}
+
+.card__button {
+  display: inline-block;
+  width: 166px;
+  height: 40px;
+  padding: 10px 10px 11px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+
+  background-color: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  transition: background-color 0.4s ease;
+
+}
+
+.card__button:hover {
+  background: #fff;
+  border-radius: 5px;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.name {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.number {
```

Comment:
what are name and number? are these separate BEM blocks? you should create BEM blocks only for the card and styles blocks